src/open_inwoner/configurations/models.py

```diff
@@ -452,6 +453,14 @@ def matomo_enabled(self):
     def siteimprove_enabled(self):
         return bool(self.siteimprove_id)
 
+    @property
+    def openid_enabled_for_admin(self):
+        return self.openid_display == "admin"
+
+    @property
+    def openid_enabled_for_regular_users(self):
+        return self.openid_display == "regular"
+
```

Comment:
These should compare to the choices on the class (eg: `OpenIdDisplayChoices.admin` instead of their literal string value).